tests/test_wps_update_metadata.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from pywps import Service
+from pywps.tests import assert_response_success
+
+from .common import client_for, TESTDATA
+from thunderbird.processes.wps_update_metadata import UpdateMetadata
+import owslib.wps
+import os
+
+@pytest.mark.online
+@pytest.mark.parametrize(
+    ("opendap"), [(TESTDATA["test_opendap"])],
+)
+@pytest.mark.parametrize(
+    ("updates"), 
+    [
+        (os.getcwd()+"/tests/metadata-conversion/updates-CLIMDEX-downscale-BCCAQ.yaml"),
+        (os.getcwd()+"/tests/metadata-conversion/updates-downscaled.yaml"),
+        (os.getcwd()+"/tests/metadata-conversion/updates-downscaled-climo.yaml"),
+        (os.getcwd()+"/tests/metadata-conversion/updates-hydromodel-dgcm.yaml"),
+        ("""
+global:
+    history: "today is a nice day"
+        """)
+    ],
+)
```

Comment:
This is great! Thank you for adding tests here. Would you be able to add the same test but with an offline resource? That way we can include it in our CI?